cookbooks/omnibus-supermarket/templates/default/logrotate.cron.erb

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+PATH=<%= @install_directory %>/embedded/sbin:$PATH
+
+command -v logrotate >/dev/null 2>&1 || exit 0
+nice -n 19 ionice -c3 logrotate <%= @var_directory %>/etc/logrotate.conf
```

Comment:
Soooo ... this depends on the system's logrotate being installed. Which was apparently the case in test. Need to add the logrotate dependency to the omnibus project and embed the command.